resources/views/layouts/links/icons.blade.php

```diff
@@ -10,6 +12,11 @@
 <link rel="apple-touch-icon" sizes="120x120" href="/images/favicon/apple-touch-icon-120x120.png" />
 <link rel="apple-touch-icon" sizes="144x144" href="/images/favicon/apple-touch-icon-144x144.png" />
 <link rel="apple-touch-icon" sizes="152x152" href="/images/favicon/apple-touch-icon-152x152.png" />
-<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon-180x180.png" />
+<link rel="apple-touch-icon" sizes="180x180" href="/images/favicon/apple-touch-icon.png" />
+<link rel="shortcut icon" href="/images/favicon/favicon.ico" />
+<link rel="manifest" href="/images/favicon/site.webmanifest" />
 
-@if (isset($campaign))<meta name="apple-mobile-web-app-title" content="{{ $campaign->name }}">@endif<meta name="apple-mobile-web-app-status-bar-style" content="#40479e">
+
+
+@if (isset($campaign))<meta name="apple-mobile-web-app-title" content="{{ $campaign->name }}">@else<meta name="apple-mobile-web-app-title" content="Kanka" />@endif
```

Comment:
We don't hardcode Kanka as the app name for people self hosting. Use config('app.name') instead.
The whole if/else can also be merged into a single line